candidate-standard/abstract_tests/collections/ATS_rc-resolutionx-definition.adoc

```diff
@@ -0,0 +1,19 @@
+[[ats_collections_rc-resolutionx-definition]]
+[width="90%",cols="2,6a"]
+|===
+^|*Abstract Test {counter:ats-id}* |*/ats/collections/rc-resolutionx-definition*
+^|Test Purpose |Validate that the resolutionx query parameters are constructed correctly.
```

Comment:
Should `resolutionx` be formatted with back ticks since it's a query parameter?

Reply:
If you say so - just discovered the magic properties of GitHub encompass not just # but also @ in all sorts of places. Need to figure out back-tick on my keyboards.